src/components/CIPipelineN/CustomInputVariableSelect.tsx

```diff
@@ -198,14 +198,16 @@ function CustomInputVariableSelect({ selectedVariableIndex }: { selectedVariable
             )
         }
     }
-
+     
     const ValueContainer = (props) => {
         let value = props.getValue()[0]?.label
+        const flag = !props.selectProps.menuIsOpen
         return (
             <components.ValueContainer {...props}>
                 <>
-                    {!props.selectProps.menuIsOpen &&
-                        (value ? `${value}` : <span className="cn-5">Select source or input value</span>)}
+                    {value ? `${value}` : flag && (
```

Comment:
{value ? `${value}` : flag  here we can directly use value without interpolation